libs/components/lookup/src/lib/modules/lookup/lookup.component.ts

```diff
@@ -315,10 +316,6 @@ export class SkyLookupComponent
   #idService = inject(SkyIdService);
   #logSvc = inject(SkyLogService);
   #modalService = inject(SkyModalService);
-  #openNativePicker =
-    inject(SkyModalInstance, {
-      optional: true,
-    }) || undefined;
   #openSelectionModal =
```

Comment:
Do we need to do the same thing for `#openSelectionModal`?

Reply:
Good catch. Updated.